packages/winpanda/extra/src/winpanda/bin/winpanda.py

```diff
@@ -0,0 +1,120 @@
+"""Panda package management for Windows.
+
+CLI entry point definition.
+"""
+from pathlib import Path
+import sys
+
+from docopt import docopt, DocoptExit
+
+sys.path.insert(1, f'{Path(sys.path[0]).parent}')
```

Comment:
Why is this needed? Can we structure the code so that it is not needed?

Reply:
Keeping an 'entry point' module (as well as all the other source modules) in a separate directory makes entire aplication's code base structure cleaner. .. to my mind. Code base's root would then contain only helper files, like requirements.txt, README.md, tox.ini, etc

I can move winpanda.py to the code base root, if the consideration above does not make sense for you.